src/ripple/app/consensus/RCLConsensus.h

```diff
@@ -151,11 +160,11 @@ class RCLConsensus
 
             If not available, asynchronously acquires from the network.
 
-            @param ledger The ID/hash of the ledger acquire
+            @param hash The ID/hash of the ledger acquire
```

Comment:
Not in this changeset, but on line 149 "called" is misspelled as "callled". On line 313 "believe" is misspelled as "believ".